test/Pass/syn-inst-trunc.ll

```diff
@@ -1,7 +1,7 @@
 ; REQUIRES: solver
 
 ; RUN: %llvm-as -o %t %s
-; RUN: %opt -load %pass -souper -dce %solver -souper-infer-inst -S -o - %s | %FileCheck %s
+; RUN: %opt -load %pass -souper -dce %solver -souper-infer-inst -souper-synthesis-comps=const -S -o - %s | %FileCheck %s
```

Comment:
if you don't specify the components to use, the default is to use them all-- which uses a lot of resources. but if you specify just a constant, synthesis is extremely boring and the test cast isn't very interesting. so maybe add a few more instructions to this list? just random stuff like add,sub,mul is fine.

Reply:
Sure, i can do that. I'm just not very sure *what* which test should be testing.
I.e. `test/Pass/`, are we testing here that we can synthesize `trunc`, or that we pick `trunc` and not some other alternative?
Like i said, it would be awesome to have an overview.